src/fping.c

```diff
@@ -2992,6 +3036,7 @@ void usage(int is_error)
     fprintf(out, "   -C, --vcount=N     same as -c, report results (not stats) in verbose format\n");
     fprintf(out, "   -d, --rdns         show targets by name (force reverse-DNS lookup)\n");
     fprintf(out, "   -D, --timestamp    print timestamp before each output line\n");
+    fprintf(out, "       --timestamp-format=FORMAT  show timestamp using the given format (-D are required): [ctime|iso|reltime]\n");
```

Comment:
`-D is required` or `-D required` to shorten it.

Reply:
To shorten it a bit more:

show timestamp in the given format (-D required): ctime|iso|reltime\n

So maybe wrapping can be avoided.

Reply:
Thanks for the feedback and the suggestion.
I have adopted the change and hope the rest of the code is also ok